### PR TITLE
Unfinished challenge

### PR DESCRIPTION
**Pull request by tommasobratto**

### CODE REVIEW

File: gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module.exports = function(grunt) {
+
+  grunt.initConfig( {
+    jshint: {
```
Comment (costassarris) [inline gruntfile.js:4]: grunt is returning some linting issues

File: test/todo-protractor-spec.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// following Protractor's tutorial
+
+describe('todo homepage', function() {
+
+  it('should initialize with a form and no tasks listed', function() {
+    browser.get('http://localhost:3000')
+  });
+});
```
Comment (dbugsy) [inline test/todo-protractor-spec.js:7]: Not actually testing anything here - maybe start by expecting the title to be correct?

**Comment by dbugsy** [comment]
:construction: 

Hi Tommaso, looks like you haven't had a chance to finish this just yet. What you have done so far looks good, let us know when it is complete.

**Comment by tommasobratto** [comment]
Hi Dan, I recently made a new repo to restart the challenge, I will let you
know when I'll have it finished, right now it is some sort of MVP and has
only unit tests, I plan to add something more to it.

2015-03-27 16:54 GMT+00:00 Daniel Le Dosquet - Bergquist <
notifications@github.com>:

**Pull request closed without merging**